aten/src/ATen/native/hip/ck_kernels/fp8_rowwise_kernel_manifest.h

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <cstdlib>
+
+#include <ATen/ATen.h>
+
+// Tiny tile kernel for small shapes.
```

Comment:
I think these ones need to be in a pytorch specific namespace to not clash with fbgemm but I haven't ran nm to verify but there could be naming clashes

Reply:
🤦 I finally understood what you were asking for.  Done.  Functions put behind a namespace to avoid clashes with fbgemm symbols.